storage/provider/managedfs_test.go

```diff
@@ -116,25 +121,55 @@ func (s *managedfsSuite) TestCreateFilesystemsNoBlockDevice(c *gc.C) {
 	c.Assert(results[0].Error, gc.ErrorMatches, "backing-volume 0 is not yet attached")
 }
 
+const testMountPoint = "/in/the/place"
+
 func (s *managedfsSuite) TestAttachFilesystems(c *gc.C) {
-	s.testAttachFilesystems(c, false, false)
+	nonRelatedFstabEntry := "/dev/foo /mount/point stuff"
+	err := ioutil.WriteFile(filepath.Join(s.fakeEtcDir, "fstab"), []byte(nonRelatedFstabEntry), 0644)
+	c.Assert(err, jc.ErrorIsNil)
+
+	mtabEntry := fmt.Sprintf("%s %s other mtab stuff", "/dev/sda1", testMountPoint)
```

Comment:
Would be a bit clearer with  `/dev/sda1` in the format string rather than subbed in.

Reply:
ok